packages/sdk/contractkit/src/kit.ts

```diff
@@ -146,9 +147,9 @@ export class ContractKit {
         const addresses = Object.values(eachTokenAddress)
         const configContractWrapper = await this.contracts.getContract(contract)
         if (humanReadable && 'getHumanReadableConfig' in configContractWrapper) {
-          return configContractWrapper.getHumanReadableConfig(addresses)
+          return await configContractWrapper.getHumanReadableConfig(addresses)
         } else if ('getConfig' in configContractWrapper) {
-          return configContractWrapper.getConfig(addresses)
+          return await configContractWrapper.getConfig(addresses)
```

Comment:
actual fix to backport to master, errors weren't caught at line 157, so the end user wouldn't know which contract call failed